src/coreclr/jit/gentree.cpp

```diff
@@ -20372,6 +20377,13 @@ GenTree* Compiler::gtNewSimdCmpOpNode(genTreeOps  op,
                     intrinsic = NI_AVX2_CompareEqual;
                 }
             }
+            else if (simdSize == 64)
+            {
+                assert(compIsaSupportedDebugOnly(InstructionSet_AVX512F));
+                assert(compIsaSupportedDebugOnly(InstructionSet_AVX512BW));
+                assert(compIsaSupportedDebugOnly(InstructionSet_AVX512DQ));
```

Comment:
With the "enable EVEX by default" PR, I've change the JIT to require all 5 "baseline" AVX512 ISAs or no AVX512 is available, so you can probably simplify this to
```suggestion
                assert(IsBaselineVector512IsaSupportedDebugOnly());
```

Reply:
Done